### PR TITLE
http status

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

**Comment by tannernelson** [comment]
Added the source and additional motivation

**Comment by paulofaria** [comment]
LGTM :rocket: ?

**Comment by paulofaria** [comment]
oh just a min

**Comment by tannernelson** [comment]
The source needs the status codes still

**Comment by paulofaria** [comment]
yeah! done.

**Comment by paulofaria** [comment]
Feel free to comment. I tried the most flexible possible.

File: HTTPStatus.swift [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        case .Conflict:                      return "Conflict"
+        case .Gone:                          return "Gone"
+        case .LengthRequired:                return "Length Required"
+        case .PreconditionFailed:            return "Precondition Failed"
+        case .RequestEntityTooLarge:         return "Request Entity Too Large"
+        case .RequestURITooLong:             return "Request URI Too Long"
+        case .UnsupportedMediaType:          return "Unsupported Media Type"
+        case .RequestedRangeNotSatisfiable:  return "Requested Range Not Satisfiable"
+        case .ExpectationFailed:             return "Expectation Failed"
+        case .ImATeapot:                     return "I'm A Teapot"
```
Comment (tannernelson) [inline HTTPStatus.swift]: wtf lol

**Comment by tannernelson** [comment]
That's a lot of code, but the public interface looks clean.

**Comment by LoganWright** [comment]
:shipit:

**Pull request merged**